github/enterprise_actions_runners.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// CreateRegistrationToken creates a token that can be used to add a self-hosted runner.
+//
+// GitHub API docs: https://docs.github.com/en/rest/reference/actions/#create-a-registration-token-for-a-repository
+func (s *EnterpriseService) CreateRegistrationToken(ctx context.Context, enterprise string) (*RegistrationToken, *Response, error) {
+	u := fmt.Sprintf("enterprises/%v/actions/runners/registration-token", enterprise)
```

Comment:
When I follow the link on line 15, I see an endpoint like this:

```
POST /repos/{owner}/{repo}/actions/runners/registration-token
```

not the endpoint I see on line 17.  Why is that?